multinet/uploaders/csv.py

```diff
@@ -26,8 +28,11 @@ def validate_csv(rows):
             else:
                 uniqueKeys.add(key)
 
+        if "?" in uniqueKeys:
+            data_errors.append({"error": "encoding", "detail": "utf-8"})
```

Comment:
Is this residual testing code? Or is this what utf8 sees as an unsupported character?

Reply:
Looks like that's just how libreoffice saved it. I'm looking into this more right now

Reply:
I would remove this check, since technically, `?` is a perfectly valid key. If your system is spitting these out in error, and there are multiple of them, then the uniqueness check will show you that, and if it only happens to spit out one, then it might pass validation but it's not really Multinet's problem to detect in that case.

Reply:
Looks like it's actually a problem on decoding if there are characters that can't be parsed. I move the error down to there in the next commit.